src/components/filter/row/MultipleStringFIlterRow.tsx

```diff
@@ -44,6 +45,7 @@ export default function MultipleStringFilterRow({ config }: MultipleStringFilter
 
 	const valueBubbleOnChangeFor = (index: number) => (nextValue: string) => {
 		config.setValues(replaceByIndex(config.values, index, nextValue));
+		focusBubbleOrInput(index + 1);
```

Comment:
What does it do?

Reply:
removed it. 
the idea was that it would allow us to target autocompleteInput after we press 'Tab', but it doesn't accomplish that.